test/integration-serial/serial_task_run_test.rb

```diff
@@ -28,4 +28,57 @@ def test_run_without_verify_result_fails_if_pod_was_not_created
       "Kubeclient::HttpError: Pod with same name exists"
     ], in_order: true)
   end
+
+  # We can't ensure that all the metrics are specific test when they are running in parallel
+  # so we run them serially
+  def test_failure_statsd_metric_emitted
+    deploy_task_template
+    task_runner = build_task_runner
+
+    result = false
+    metrics = capture_statsd_calls do
+      result = task_runner.run(run_params.merge(args: ["/not/a/command"]))
```

Comment:
Can we use a failure scenario that fails validations and doesn't launch the pod at all? The idea is to keep these serial tests as quick as possible, and this one doesn't really need to launch anything to be valid.